test/runner/command.js

```diff
@@ -0,0 +1,145 @@
+import yargs from "yargs/yargs";
+import { browsers } from "./browsers.js";
+import { getPlan, listBrowsers, stopWorkers } from "./browserstack/api.js";
+import { buildBrowserFromString } from "./browserstack/buildBrowserFromString.js";
+import { modules } from "./modules.js";
+import { run } from "./run.js";
+import { jqueryMigrate } from "./jquery-migrate.js";
+import { jquery } from "./jquery.js";
+
+const argv = yargs( process.argv.slice( 2 ) )
+	.version( false )
+	.strict()
+	.command( {
+		command: "[options]",
+		describe: "Run jQuery Migrate tests in a browser"
+	} )
+	.option( "jquery-migrate", {
+		type: "array",
+		choices: jqueryMigrate,
+		description: "Versions of jQuery Migrate to test." +
+			"Defaults to the minified version in the dist/ folder.",
+		default: [ "min" ]
+	} )
+	.option( "jquery", {
+		type: "array",
+		choices: jquery,
+		description: "Versions of jQuery to test.",
+		default: [ "3.x-git.min" ]
+	} )
+	.option( "module", {
+		alias: "m",
+		type: "array",
+		choices: modules,
+		description:
+			"Run tests for a specific module. " +
+			"Pass multiple modules by repeating the option. " +
+			"Defaults to all modules."
+	} )
+	.option( "browser", {
+		alias: "b",
+		type: "array",
+		choices: browsers,
+		description:
+			"Run tests in a specific browser." +
+			"Pass multiple browsers by repeating the option." +
+			"If using BrowserStack, specify browsers using --browserstack.",
+		default: [ "chrome" ]
+	} )
+	.option( "headless", {
+		alias: "h",
+		type: "boolean",
+		description:
+			"Run tests in headless mode. Cannot be used with --debug or --browserstack.",
+		conflicts: [ "debug", "browserstack" ]
+	} )
+	.option( "concurrency", {
```

Comment:
The `esm` option should be supported here as well, just like in Core.

Reply:
I don't think it should. Core sets the version of one library. It uses top level options and parameters to set the core version. Conversely, migrate always sets the versions of two distinct libraries. Instead, esm has been supported in migrate by passing `esmodules` as a *value* to the `plugin` URL parameter, and is supported here as a value of the `jquery-migrate` option.

Reply:
Good point. No change needed then.